### ISSUE TITLE
Love your work!

### ISSUE THREAD

**Issue by ArjixWasTaken** [hellflame/argparse#8]
I seriously love this fork, it is maintained and has more features than the original repo.

❤️ ❤️ ❤️

You sir earned my star.

**Comment by hellflame** [comment]
Thank you for loving this repo ~
It is my goal to make argparsing better

(but actually, this is not a fork)

**Comment by ArjixWasTaken** [comment]
OOF, my bad, google search brought me to another repo (and that repo did not have any indicator of being a fork)
and I found you at an issue of that repo that linked to this repo, so I assumed it was a fork

**Comment by hellflame** [comment]
> Ok, gave it a closer look, do you just share the same name and API interface? https://github.com/akamensky/argparse

Well, I tried https://github.com/akamensky/argparse before I create this repo (it's true the two repos are very connective), but there's a lot feature missing.
Then I checked it's code, but it's too hard to upgrade, and here comes this repo, code from scratch.

There are API interface diffs between two repos. Some argument types are not implemented, more features are enabled, configs about ParserConfig & ArgumentOption are carefully distinguished ... There's more difference and more thoughts about argparsing.
